website/addons/evernote/views.py

```diff
@@ -0,0 +1,215 @@
+from flask import request
+import httplib as http
+
+from framework.auth.decorators import must_be_logged_in
+from framework.exceptions import HTTPError, PermissionsError
+
+from website.addons.base import generic_views
+from website.addons.evernote import utils
+from website.addons.evernote.serializer import EvernoteSerializer
+from website.oauth.models import ExternalAccount
+from website.util import permissions
+from website.project.decorators import (
+    must_be_contributor_or_public,
+    must_have_addon,
+    must_be_addon_authorizer,
+    must_not_be_registration,
+    must_have_permission
+)
+
+import ENML2HTML
+
+import logging
+logger = logging.getLogger(__name__)
+
+SHORT_NAME = 'evernote'
+FULL_NAME = 'Evernote'
+
+@must_be_logged_in
+def evernote_get_user_settings(auth):
```

Comment:
Many of these view functions can be replaced with `generic_views`. This should be replaced with `generic_views.account_list`